README.md

```diff
@@ -94,15 +92,14 @@ The guidelines described below are based on:
 
   *Why?*: Keeps consistency with AngularJS 1.x style guidelines.
 
-  ### Example
-
-  **CORRECT**
   ```
+  /* CORRECT */
```

Comment:
Don't you think the above is more obvious and clear which is the correct example and which is the wrong one?

With comment, depending on the code theme it may get harder to notice.